<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simplified Stress Measurement - LakeShore Model 372</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startbutton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Set Settings
 And Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="abortbutton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="interpolatebutton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpolate CSV File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plotbutton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot from Interpolated 
CSV File (Using PyPlot)</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="taskprogressbar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>80</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="totalprogressbar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Est. Task Process:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Est. Total Progress:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Task:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CurrentTask">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>231</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Quick Settings</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Note: Make sure to match settings of the
 LakeShore software for channel config. 
Also make sure to enable the channels and
 enable excitation manually.</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>211</width>
       <height>211</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Heater</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>201</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="1">
         <widget class="QLineEdit" name="htrmaxcurrent"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="htrresistance"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Final %</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Resistance</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Delta %</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="htrdelta"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="htrfinal"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Init %</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="htrinit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Max Current:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>Range:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="htrrange"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>MCThermo</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>201</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QLineEdit" name="mcthermosettle"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="mcthermochannel"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Dwell Time</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Settle Time</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Pause Time</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="mcthermodwell"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="mcthermopause"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Sample 1</string>
      </attribute>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Excitation Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>201</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="1">
         <widget class="QLineEdit" name="sample1settle"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sample1channel"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Dwell Time</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Settle Time</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Pause Time</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="sample1dwell"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="sample1pause"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="sample1description"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Sample 2</string>
      </attribute>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Excitation Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>201</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="1">
         <widget class="QLineEdit" name="sample2settle"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sample2channel"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Dwell Time</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Settle Time</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Pause Time</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="sample2dwell"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="sample2pause"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="sample2description"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Sample 3</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>201</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="4" column="1">
         <widget class="QLineEdit" name="sample3settle"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sample3channel"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>Dwell Time</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>Settle Time</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>Pause Time</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="sample3dwell"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="sample3pause"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="sample3description"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>211</width>
       <height>155</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="6" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Switching Time (s):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="thermalizationtime"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="switchingtime"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Thermalization Time (s):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="serialportopen">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="serialportclose">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serialportport">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="serialportbaud">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Serial Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Baud Rate:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="measurementpasses"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string># of Passes/Htr setting:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="rundescription">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description to Append to Filename:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="interpolatebutton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save new
config.ini</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
